apps/src/templates/feedback/LevelFeedbackEntry.jsx

```diff
@@ -63,31 +118,15 @@ export default class LevelFeedbackEntry extends Component {
       performance
     } = this.props.feedback;
 
-    const {expanded, commentHeight} = this.state;
-
-    const seenByStudent = seen_on_feedback_page_at || student_first_visited_at;
-
     const commentExists = comment.length > 2;
 
-    // These heights ensure that the initial line of the comment will be visible, and a 'sneak peak' of the second line for long comments.
-    let baseHeight = 72;
```

Comment:
I realized this whole baseHeight thing was just not necessary, so excited to remove this